src/main/kotlin/com/atlassian/performance/tools/ssh/PerformanceDefaultConfig.kt

```diff
@@ -0,0 +1,25 @@
+package com.atlassian.performance.tools.ssh
+
+import net.schmizz.sshj.DefaultConfig
+import net.schmizz.sshj.transport.random.BouncyCastleRandom
+import net.schmizz.sshj.transport.random.JCERandom
+import net.schmizz.sshj.transport.random.Random
+import net.schmizz.sshj.transport.random.SingletonRandomFactory
+import net.schmizz.sshj.common.Factory
+
+internal class PerformanceDefaultConfig : DefaultConfig() {
+    companion object {
+        val bcFactory = MemoizingFactory(BouncyCastleRandom.Factory())
+        val jceFactory = MemoizingFactory(JCERandom.Factory())
```

Comment:
So the clue is to reuse RNG. I presume this reduces likelihood of some kind of collisions? Also, does it mean that `SSHClient` instances should be reused? We didn't reuse them by default, because they were not documented and we saw that it `extends SocketClient`, which has a `Socket` field. I inferred it's stateful per connection.

Reply:
We don't need to reuse RNG, but reuse of secure random generators is a generally recommended pratice (not just here but in general)